src/components/Utils/PrivateRouteShelter.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```

Comment:
Thanks for adding this! Good way to separate normal protected routes and Shelter protected routes